python/tchannel/thead.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2015 Uber Technologies, Inc.
```

Comment:
In that case all the 'field' in the transport header should have their own file? 
name	req	res	description
as	Y	Y	the Arg Scheme
cas	Y	N	Claim At Start
caf	Y	N	Claim At Finish
cn	Y	N	Caller Name
re	Y	N	Retry Flags
se	Y	N	Speculative Execution
fd	Y	Y	Failure Domain

I think move them in central place like 'transport header ' file would be easy to manage.